docs/guide/robot_sbc/setup_jetson_nano.md

```diff
@@ -2,36 +2,103 @@
+These are the supported versions:
 
-Once you're done with the setup, ssh into your vehicle. Use the the terminal for Ubuntu or Mac. [Putty](https://www.chiark.greenend.org.uk/~sgtatham/putty/latest.html) for windows.
+| Jetson      | Jetpack | Python | Donkey   | Tensorflow |
+|-------------|---------|--------|----------|------------|
+| Nano        | 4.5.2   | 3.6    | <= 4.5.X | 2.3.1      |   
+| Nano        | 4.6.2   | 3.9    | >= 5.X   | 2.9        |
+| Xavier/Orin | 4.6.2   | 3.8    | >= 5.X   | 2.9        | 
 
-## Step 2: Free up the serial port (optional. Only needed if you're using the Robohat MM1)
 
+Then [Create your Donkeycar Application](/guide/create_application/)
+
+
+## Installation for Donkey Car <= 4.5.X
+
+Instructions for the latest stable release 4.5.X.
+
+* [Step 1a: Flash Operating System](#step-1a-flash-operating-system)
+* [Step 2a: Free up the serial port](#step-2a-free-up-the-serial-port-optional-only-needed-if-youre-using-the-robohat-mm1)
+* [Step 3a: Install Dependencies](#step-3a-install-system-wide-dependencies)
+* [Step 4a: Setup Python Environment](#step-4a-setup-python-environment)
+* [Step 5a: (Optional) Install PyGame for USB camera](#step-5a-optional-install-pygame-for-usb-camera)
+
+
+### Step 1a: Flash Operating System
+
+
+These instructions work for Jetpack 4.5.1. 
+
+* If you have a 4gb Jetson Nano then download Jetpack 4.5.1 from Nvidia
+  here; [jetson-nano-jp451-sd-card-image.zip](https://developer.nvidia.com/embedded/l4t/r32_release_v5.1/r32_release_v5.1/jeston_nano/jetson-nano-jp451-sd-card-image.zip)
+* If you have a 2gb Jetson Nano the download Jetpack 4.5.1from Nvidia
+  here; [jetson-nano-2gb-jp451-sd-card-image.zip](https://developer.nvidia.com/embedded/l4t/r32_release_v5.1/r32_release_v5.1/jeston_nano_2gb/jetson-nano-2gb-jp451-sd-card-image.zip)
+
+This installs the official Nvidia build of Tensorflow 2.3.1; make sure you are
+using the same version of Tensorflow on your host PC if you are using one. Using
+a different version of Tensorflow to train your network may result in errors
+when you attempt to use it as an autopilot.
+
+Visit the official [Nvidia Jetson Nano Getting Started Guide](https://developer.nvidia.com/embedded/learn/get-started-jetson-nano-devkit#prepare)
+or [Nvidia Xavier NX Getting Started Guide](https://developer.nvidia.com/embedded/learn/get-started-jetson-xavier-nx-devkit).
+Work through the __Prepare for Setup__, __Writing Image to the microSD Card__,
+and __Setup and First Boot__ instructions, then return here.
+
+Once you're done with the setup, ssh into your vehicle. Use the terminal for
+Ubuntu or Mac. [Putty](https://www.chiark.greenend.org.uk/~sgtatham/putty/latest.html)
+for windows.
+
+Remove Libre Office:
+
+```bash
+sudo apt-get remove --purge libreoffice*
+sudo apt-get clean
+sudo apt-get autoremove
 ```
+
+And add a 6GB swap file:
+
+```bash
+git clone https://github.com/JetsonHacksNano/installSwapfile
+cd installSwapfile
+./installSwapfile.sh
+reboot 
```

Comment:
Received: 
```
Failed to set wall message, ignoring: Interactive authentication required.
Failed to reboot system via logind: Interactive authentication required.
Failed to open /dev/initctl: Permission denied
Failed to talk to init daemon.
```

Suggest: `sudo reboot now` instead.

Reply:
I believe this depends on how you login to the Jetson. I'll change to what you suggest. Can you confirm that the swapfile script also works without `sudo`?

Reply:
> depends on how you login to the Jetson

The earlier instructions say:

> Once you're done with the setup, ssh into your vehicle

So it depends on what you're thinking of when you say "depends"! Are you thinking of terminal in a regular keyboard/mouse/monitor session?

I think `sudo reboot` should work in either case.